js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -177,3 +202,438 @@ category=Kategorie
+defaultAdminInitiated=Standard-Lebensdauer für durch den Admin initiierte Aktionen
+defaultAdminInitiatedActionLifespanHelp=Maximale Zeit, bevor eine vom Administrator an einen Benutzer gesendete Aktionsgenehmigung abläuft. Es wird empfohlen, diesen Wert lang zu wählen, damit Administratoren E-Mails für Benutzer senden können, die derzeit offline sind. Die Standardzeitüberschreitung kann unmittelbar vor der Ausstellung des Tokens überschrieben werden.
+emailVerification=E-Mail-Verifizierung
+idpAccountEmailVerification=E-Mail-Verifizierung des IdP-Kontos
+executeActions=Ausführen von Aktionen
+clientPolicies=Client Richtlinien
+profiles=Profile
+policies=Richtlinien
+newClientProfile=Client Profil erstellen
+profilesConfigType=Konfigurieren mit\:
+policiesConfigTypes.formView=Formular
+policiesConfigTypes.jsonEditor=JSON Editor
+noClientPolicies=Keine Client Richtlinien vorhanden
+noClientPoliciesInstructions=Es sind keine Kundenrichtlinien vorhanden. Wählen Sie "Client-Richtlinie erstellen", um eine neue Client-Richtlinie zu erstellen.
+createPolicy=Client-Richtlinie erstellen
+clientPolicySearch=Client-Richtlinie suchen
+createClientPolicyError=Richtlinie konnte nicht erstellt werden aufgrund von\: {{error}}
+userProfile=Benutzerprofil
+attribute=Attribut
+attributesGroup=Attributgruppe
+allGroups=Alle Gruppen
+createAttribute=Attribut erstellen
+createGroupText=Attributgruppe erstellen
+attributeDisplayName=Anzeigename
+columnDisplayDescription=Beschreibung
+annotations=Annotationen
+hideInheritedRoles=Geerbte Rollen ausblenden
+assignRole=Rolle zuweisen
+searchByName=Suche nach Namen
+searchByRoleName=Suche nach Rollennamen
+unAssignRole=Rolle entfernen
+whatIsDefaultGroups=Was ist die Funktion von Standardgruppen?
+noDefaultGroupsInstructions=Mit Standardgruppen können Sie automatisch eine Gruppenzugehörigkeit zuweisen, wenn ein neuer Benutzer erstellt oder während des <1>Identitätsbrokerings</1> importiert wird.
+noDefaultGroups=Keine Standardgruppen definiert
+authenticationExplain=Im Bereich Authentifizierung können Sie verschiedene Arten von Anmeldeinformationen konfigurieren und verwalten.
+flows=Abläufe
+searchForFlow=Suche nach Ablauf
+createFlow=Ablauf erstellen
+authenticationCreateFlowHelp=Ablauf erstellen
+flowName=Ablaufname
+flowType=Ablauftyp
+usedBy=Vewendet von
+duplicate=Duplizieren
+setAsDefaultAction=Als Standardaktion festlegen
+authDefaultActionTooltip=Wenn diese Option aktiviert ist, wird jedem neuen Benutzer diese erforderliche Aktion zugewiesen.
+disabledOff=Deaktiviert aus
+passwordPolicy=Passwortrichtlinien
+noPasswordPolicies=Keine Passwortrichtlinien definiert
+noPasswordPoliciesInstructions=Sie haben keine Passwortrichtlinien zu diesem Bereich hinzugefügt. Fügen Sie eine Richtlinie hinzu, um loszulegen.
```

Comment:
"um zu beginnen/anzufangen"?

Reply:
I changed "um loszulegen" to "um zu beginnen".